src/core/model/uischema.ts

```diff
@@ -7,12 +7,14 @@
  */
 import { isLayout, Layout, UISchemaElement } from '@jsonforms/core';
 import { cloneDeep } from 'lodash';
+import { v4 as uuid } from 'uuid';
 
 import { calculatePath, getRoot, isPathError, PathError } from '../util/clone';
 
 export interface LinkedUISchemaElement extends UISchemaElement {
-  linkedSchemaElements?: Array<string>;
+  linkedSchemaElements?: Set<string>;
   parent?: LinkedUISchemaElement;
+  uuid?: string;
```

Comment:
shoudn't this be mandatory?

Reply:
Yes, ideally it should, but if it would be mandatory we can't assign an UISchemaElement to a LinkedUISchemaElement anymore. We would need to do a cast each time we access the `elements` of a Layout, for example. But I can change this if you want.

Reply:
no keep it like it is.

Reply:
For now this is fine I think.